common/src/main/java/tachyon/resource/ResourcePool.java

```diff
@@ -40,11 +40,21 @@
    * @param maxCapacity the maximum of resources in this pool
    */
   public ResourcePool(int maxCapacity) {
+    this(maxCapacity, new LinkedBlockingQueue<T>(maxCapacity));
+  }
+
+  /**
+   * Internal constructor that can provide an object to be used for the internal queue.
+   *
+   * @param maxCapacity the maximum of resources in this pool.
```

Comment:
The style we use is either ``The maximum of resources in this pool.`` or ``the maximum of resources in this pool`` (note the different capitalization and punctuation)